First test task 0-module:

ShaludkinaNatalia opened the pull request:
Необходимо написать функцию, которая суммирует два числа

sum(1,1) === 2
sum(2,5) === 7

ShaludkinaNatalia committed:
First test task 0-module:

Необходимо написать функцию, которая суммирует два числа

sum(1,1) === 2
sum(2,5) === 7

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

```diff
Допишите функцию валидации имени пользователя?
- если меньше 4 симовлов то false
- если содержит пробелы, то false
- если имя корректное то true
```
```diff
Первая задача
+ 1+1=2
```
```diff
Функция pow(x,n)
- при возведении 3 в 4ю степень равен 81
- при возведении 2 в 3ю степень результат 8
```

ShaludkinaNatalia committed:
First test task 0-module:

Необходимо написать функцию, которая суммирует два числа

sum(1,1) === 2
sum(2,5) === 7

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

```diff
Допишите функцию валидации имени пользователя?
- если меньше 4 симовлов то false
- если содержит пробелы, то false
- если имя корректное то true
```
```diff
Функция pow(x,n)
- при возведении 2 в 3ю степень результат 8
- при возведении 3 в 4ю степень равен 81
```
```diff
Первая задача
+ 1+1=2
```

ShaludkinaNatalia committed:
1-task 1-module:

Необходимо написать функцию, которая суммирует два числа

sum(1,1) === 2
sum(2,5) === 7

2-task 1-module:

Написать функцию валидации имени пользователя

jsru commented:
Поздравляю, все задачи решены абсолютно верно
```diff
Функция pow(x,n)
+ при возведении 3 в 4ю степень равен 81
+ при возведении 2 в 3ю степень результат 8
```
```diff
Допишите функцию валидации имени пользователя?
+ если содержит пробелы, то false
+ если имя корректное то true
+ если меньше 4 симовлов то false
```
```diff
Первая задача
+ 1+1=2
```
